src/vtlengine/Operators/Numeric.py

```diff
@@ -1,11 +1,13 @@
+import _random
```

Comment:
Where does this come from? Is it the same as random package? Why using _random?

Reply:
_random contains the random class prototypes, so for class inheritance it is better to use _random as it generates fewer errors